### ISSUE TITLE
Program - QA Program - Базовые программы для QA - Jira: Lessons have wrong links

### ISSUE THREAD

**Issue by MykhailoBorovikov** [scholokov/qax-portal#229]
Lesson have wrong links and won't opens

**Preconditions:**
Opened portal.qax-camp.com.ua
Be logged in

**STR:**

1. Open Program - QA Program
2. Scroll down to Базовые программы для QA
3. Open Jira course
4. Open lesson Jira - Создание задачи/бага

**Actual:**
Lessons have wrong links and won't opens

**Expected:**
Lessons have correct links and opens
![Screenshot_47](https://user-images.githubusercontent.com/104158199/168417012-652e9d16-14c6-4578-b852-17ace1d169dd.png)

**Comment by NataliaKutashenko** [comment]
1. Open portal.qax-camp.com.u -missing letter "a" at the end of the URL.
2. Open - means to perform an act, as long as your preconditions mean you have already done the action. F.E. "portal.qax-camp.com.ua opened."

**Comment by MykhailoBorovikov** [comment]
> 1. Open portal.qax-camp.com.u -missing letter "a" at the end of the URL.
> 2. Open - means to perform an act, as long as your preconditions mean you have already done the action. F.E. "portal.qax-camp.com.ua opened."

Thx

**Comment by scholokov** [comment]
@GeorgiiZ90 
надо передобавить топики в курс
раньше они находились в формате "уроков", а сейчас в формате "топиков"

**Comment by GeorgiiZ90** [comment]
Fixed

**Comment by MykhailoBorovikov** [comment]
Verification is blocked by issue #230
